DependencyInjection/SyliusRbacExtension.php

```diff
@@ -73,16 +73,13 @@ public function prepend(ContainerBuilder $container)
      */
     private function prependDoctrineCache(ContainerBuilder $container)
```

Comment:
`prependDoctrineCache` is left unused?

Reply:
Yes, as the cache is configured in app/config.yml where previously the parameter was defined